public/app/core/utils/UserProvider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import { getBackendSrv } from '@grafana/runtime';
+
+export interface UserAPI {
+  changePassword: (ChangePassword: ChangePasswordFields) => void;
+}
+
+export interface ChangePasswordFields {
+  oldPassword: string;
+  newPassword: string;
+  confirmNew: string;
+}
+
+export interface Props {
+  children: (api: UserAPI) => JSX.Element;
```

Comment:
Next step for this one is to expose a current state of the API operation, whether it's pending, loading or it failed.

Reply:
I think this would be a great issue for after all of the user profile has been converted.